guides/hiring/remote_test/language_agnostic_marking_guide.md

```diff
@@ -0,0 +1,32 @@
+# Marking guide
+
+70 marks in total. There are no half marks. 80% (56 marks) required to pass.
+
+ - [2] The timeline appeared on the homepage
+ - [2] The form appeared on the homepage
+ - [3] The form displayed specific error if the message is empty
+ - [3] The form displayed specific error if the message is more than 150 characters
+ - [3] The timeline displayed message if form submitted without errors
+ - [3] The page displayed a message if form submitted without errors
+ - [2] The timeline appeared below the form
+ - [3] The timeline is ordered in descending chronological order
+ - [2] Each item in the timeline contains a message
+ - [2] Each item in the timeline contains the time of submission
+ - [2] Each item in the timeline contains a way to delete the post
+ - [3] Delete functionality is not triggered by a GET request
+ - [3] Message can contain newlines, new lines are rendered
+ - [3] Time is displayed in format of "20 minutes ago", "5 days ago"
+ - [3] Clicking delete button results in item being deleted
+ - [3] Clicking delete button results in message being displayed
+ - [3] Code is version controlled
+ - [2] Feature test: successful submission are passing
+ - [2] Feature test: unsuccessful submission are passing
+ - [2] Feature test: message display are passing
+ - [2] Feature test: message deletion are passing
+ - [3] Unit test: both validation rules are tested and passing
+ - [2] A nicely readable assertion library is used
+ - [3] README contains instructions for running application
+ - [3] README contains instructions for running tests
+ - [3] A linter is set up and passing for all languages used in the project
+ - [3] There are browser based tests using selenium or similar, all tests pass
```

Comment:
I think it's quite good to show that a dev knows how to use selenium... You still need a sprinkling of E2E smoke tests on top, generally around the areas of a system that add the most value. Where it becomes a problem is when you use slow E2E tests to TDD out actual logic. IMO that's not what Selenium is for.